velox/type/Variant.h

```diff
@@ -31,7 +31,7 @@
 namespace facebook::velox {
 
 // Constant used in comparison of REAL and DOUBLE values.
-constexpr double kEpsilon{0.00001};
+constexpr double kEpsilon{0.0000001};
```

Comment:
The old kEpsilon value is too large, which leads to one float map key incorrectly match a different float map key during the comparison of maps.